.github/workflows/cd.yml

```
@@ -0,0 +1,122 @@
+
+      - name: Add EC2 Host to known_hosts
+        run: |
+          ssh-keyscan -H ${{ secrets.EC2_HOST }} >> ~/.ssh/known_hosts
+
+      - name: Deploy to server
+        run: |
+          BRANCH="${{ github.head_ref || github.ref_name }}"
+          echo "Deploying branch: $BRANCH"
+          ssh ${{ secrets.EC2_USER }}@${{ secrets.EC2_HOST }} << 'EOF'
+            set -e
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)